lib/Doctrine/DBAL/Schema/SchemaException.php

```diff
@@ -180,4 +182,14 @@ public static function alterTableChangeNotSupported($changeName)
             sprintf("Alter table change not supported, given '%s'", $changeName)
         );
     }
+
+    public static function viewAlreadyExists(string $viewName)
+    {
+        return new self("The view '" . $viewName . "' already exists.", self::VIEW_ALREADY_EXISTS);
```

Comment:
Let's try using double quotes for human-readable messages. I.e. `The view "foo" already exists`, not `The view 'foo' already exists`. It's purely an aesthetic/typography wish.